tests/pytest.ini

```diff
@@ -4,6 +4,8 @@ markers =
     sanity: marks tests as sanity
     dut: mark tests with dut which need dut
     device: marks the tests as device configuration test
+    l1_manual: marks test as layer1 manual which needs to be run separately for specific Novus card
+    serial
```

Comment:
Add a new marker called `raw`. We can later decide what name is more suitable.
The purpose is to add it to the test scripts which currently can be run against athena (since it doesn't support devices and PFC).